<?xml version="1.0"?>
<launch>
  <arg name="pacmod_vehicle_type" default="LEXUS_RX_450H" />
  <arg name="controller_type" default="LOGITECH_F310"/>
  <arg name="is_pacmod_3" default="true" />

  <include file="$(find lexus_base)/launch/tf_static.launch" />
  <include file="$(find lexus_base)/launch/tf_duro_front.launch" />
  <node pkg="lexus_base" type="translator_joy_vel.py" name="joy_to_pacmod" output="screen" />
  <node pkg="lexus_base" type="path_and_steering" name="path_and_steering_mark" output="screen">
      <param name="pose_topic" value="/gps/duro/current_pose"/>
      <param name="path_size" value="500"/>
  </node>
  <node pkg="rviz_markers" type="lexus" name="lexus_f" output="screen">
          <param name="frame_id" value="/base_link"/>
  </node>
  
  <!-- PacMod Driver -->
  <include file="$(find vehicle_platform)/launch/platform.launch">
    <arg name="use_dbw" value="true" />
  </include>

  <param if="$(eval arg('controller_type') == 'LOGITECH_F310')" name="game_control/joy/dev" value="/dev/input/gamepad-logitech" />
  <param if="$(eval arg('controller_type') == 'HRI_SAFE_REMOTE')" name="game_control/joy/dev" value="/dev/input/gamepad-fort" />
</launch>
